src/tunserver.cpp

```diff
@@ -609,6 +611,7 @@ void c_tunserver::configure_mykey() {
 
 	// save signature and IDI publickey in tunserver
 	m_my_IDI_pub = my_IDI->m_pub;
+	m_my_IDI_pub.set_ipv6_prefix(m_ipv6_prefix);
```

Comment:
This is not a property of crypto key, what prefix/mask it has...

We need to rethink this all.

But can be ok for now to test how internal network works via galaxy42.